src/components/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { css } from 'styled-components';
+
+export const resetButton = css`
+  padding: 0;
+  margin: 0;
+  appearance: none;
+  background: 0;
+  border: 0;
+  box-shadow: none;
+`;
```

Comment:
I like this. I'm not sure about the file name, though. Maybe we can be more explicit? `shared-styles`, `css-collection`? I can't think of nice examples so we might as well leave it as `utils.ts`... it was just a thought.

Reply:
I didn't want to spend on naming that thing yet. I took some time now to think about it and `css-helpers.ts` it is now. I hope that suits you better, I like it more than the generic utils barrel